projects/managed-hubs/support.md

```diff
@@ -1,118 +1,217 @@
+
+## Support triage and resolution process
+
+Here is the process that we follow when triaging and resolving support requests.
+
+### Triage process
+
+The goal of the triage phase is to understand the Support Request, decide if it is related to an incident, and choose the appropriate resolution pathway.
+
+1. **Read and understand**. Within 8 working hours, read the support request and try to understand what action would resolve it.
+2. **Decide if there is an incident**. Determine if a request meets {term}`the definition of an incident <Incident>`.
+3. **Categorize the Support Ticket** in FreshDesk.
+4. **If an Incident**, go to [](support:incident-response).
+5. **If not an Incident**, go to [](support:non-incident-response).
+
+(support:incident-response)=
+## Incident response process
+
+Incidents are a special kind of support ticket, because they are related to degraded service that immediately impacts communities.
+We prioritize the resolution of incidents above all other kinds of work, and have a special process for tracking conversation and progress with them.
+
+Here is the process that we follow for incidents:
+
+1. **Acknowledge the incident**. Communicate with the Community Representative that there is an incident. Here is a template to get started:
+   
+   ```
+   Hello { NAME }, we have investigated this request and have concluded that
+   it is related to an incident that is causing diminished service for your
+   community.
+   
+   We believe that this incident is related to { CONTEXT HERE } and will
+   investigate further on next actions. Information about our incident
+   response process can be found [in our team support documentation](https://team-compass.2i2c.org/en/latest/projects/managed-hubs/support.html).
+ 
+   We'll prioritize resolving this incident over our other work, and
+   will communicate with you throughout our attempts to resolve it.
+   We might be in touch with requests for clarifications if needed.
+   ```
+2. **Open an incident issue**. For each {term}`Incident` we create a dedicated issue to track its progress. [{bdg-primary}`open an incident issue`](https://github.com/2i2c-org/infrastructure/issues/new?assignees=&labels=type%3A+Hub+Incident%2Csupport&template=3_incident-report.md&title=%5BIncident%5D+%7B%7B+TITLE+%7D%7D) and notify our engineering team via Slack.
+3. **If you can resolve in 30 minutes**, try doing so.
+4. **If you cannot resolve in 30 minutes**, ping our engineering team and our Project Manager in the {guilabel}`#support-freshdesk` channel so that they are aware of the incident.
+5. **Investigate and resolve the incident**. Follow the structure of the incident issue you opened in the step above.
+6. **Communicate every few hours**. Provide periodic updates to communities as we attempt to resolve the incident. Here is a template to get started:
```

Comment:
Who does this? It could be the support steward but I think it should also be ok to pass it on to an 'incident commander' for a particular incident.

Reply:
I think for this we'd need to define a new role, and incorporate that role into this process more generally. Do you think now is the right time to do that? (I am happy to put in the cycles to do this if you think it's a good time)

Reply:
ya I think now is the right time to do that. Wanna get on a call to hash it out? Doesn't need to be part of this PR tho